### ISSUE TITLE
 Lint/SafeNavigationChain should allow presence_in (ActiveSupport) method

### ISSUE THREAD

**Issue by swordray** [rubocop/rubocop#11964]
## Is your feature request related to a problem? Please describe.

If ActiveSupport is required, the `.presence_in` method should be permitted in a safe-navigation chain:

```
foo&.to_sym.presence_in([:bar, :baz])
```

Currently this raises the error: `Lint/SafeNavigationChain: Do not chain ordinary method call after safe navigation operator.`

## Additional context

Rubocop 1.52.1
